.size-limit.js

```diff
@@ -87,12 +87,12 @@ module.exports = [
     name: 'The main docs bundle',
     webpack: false,
     path: main.path,
-    limit: '193 KB',
+    limit: '199 KB',
   },
   {
     name: 'The docs home page',
     webpack: false,
     path: `.next/static/${buildId}/pages/index.js`,
-    limit: '9.8 KB',
+    limit: '10 KB',
```

Comment:
I'm curious how they achieved 26% reduction when it increased our bundle size.

Reply:
Marketing.